source/about/certifications-and-compliance.rst

```diff
@@ -84,10 +84,10 @@ Summary Table
 +-----------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------+
 | Mattermost Product                            | Export Control Classification Number (ECCN)                                                                                                     |
 +===============================================+=================================================================================================================================================+
-| Mattermost Enterprise Edition E10             | `ECCN 5D002 <https://www.bis.doc.gov/index.php/documents/regulations-docs/federal-register-notices/federal-register-2014/951-ccl5-pt2/file>`__  |
```

Comment:
For line 87 & 90, let's check with Jamie that we can just change the product names.   I am not familiar with the export control classification process.

Reply:
I've reverted the two ECCN plan name changes.

Reply:
Reverted these specific changes